x-pack/plugins/apm/server/routes/services.ts

```diff
@@ -662,6 +663,55 @@ const serviceProfilingStatisticsRoute = createApmServerRoute({
   },
 });
 
+const serviceAlertsRoute = createApmServerRoute({
+  endpoint: 'GET /api/apm/services/{serviceName}/alerts',
+  params: t.type({
+    path: t.type({
+      serviceName: t.string,
+    }),
+    query: t.intersection([
+      rangeRt,
+      environmentRt,
+      t.type({
+        transactionType: t.string,
+      }),
+    ]),
+  }),
+  options: {
+    tags: ['access:apm'],
+  },
+  handler: async ({ context, params, apmRuleRegistry }) => {
+    const alertsClient = context.alerting.getAlertsClient();
+
+    const {
+      query: { start, end, environment, transactionType },
+      path: { serviceName },
+    } = params;
+
+    const apmRuleRegistryClient = await apmRuleRegistry.createScopedRuleRegistryClient(
+      {
+        alertsClient,
+        context,
+      }
+    );
```

Comment:
If this is repeated many places perhaps we should make a small helper?

```suggestion
    const {
      query: { start, end, environment, transactionType },
      path: { serviceName },
    } = params;

	const apmRuleRegistryClient = await getApmRuleRegistryClient(context, apmRuleRegistry);
```

Reply:
Just this one for now.